mongo_connector/doc_managers/formatters.py

```diff
@@ -96,6 +96,8 @@ def transform_value(self, value):
             return value
         elif isinstance(value, datetime.datetime):
             return value
+        elif isinstance(value, bson.dbref.DBRef):
+            return value.id
```

Comment:
Yes, you are correct, this change does not cover **database** and **collection** value, but only the **id** value.
At least, it supports DBRef data from MongoDB, and it can be used by complex search like "join" in Solr.
In order to cover database and collection value, we need to modify function _transform_element_ and _format_document_ which I'm working on it.